docs/schema.graphql

```diff
@@ -1,1060 +0,0 @@
-schema {
-  query: RootQueryType
-}
-
-type Account {
-  account_udts(input: AccountChildAccountUdtsInput = {page: 1, page_size: 20, sort_type: DESC}): [AccountUdt]
-  eth_address: String
-  id: Int
-  nonce: Int
-  script: Json
-  script_hash: String
-  short_address: String
-  smart_contract: SmartContract
-  token_transfer_count: Int
-  transaction_count: Int
-  type: AccountType
-}
-
-input AccountChildAccountUdtsInput {
-  """
-  argument: the page of query result, the relations of postgres offset: offset = (page - 1) * page_size
-  
-  default: 1
-  """
-  page: Int = 1
-
-  """
-  argument: the page_size of query result, it's the same of postgres limit
-  
-  default: 20
-  """
-  page_size: Int = 20
-
-  """
-  argument: the sort of type by custom condition
-  
-  default: desc
-  """
-  sort_type: SortType = DESC
-}
-
-type AccountCkb {
-  address_hash: String
-  balance: Decimal
-}
-
-input AccountCkbsInput {
-  address_hashes: [String] = []
-}
-
-input AccountInput {
-  """
-  argument: account address(eth_address or short_address)
-  example-1: "0x15ca4f2165ff0e798d9c7434010eaacc4d768d85"
-  example-2: "0xc5e133e6b01b2c335055576c51a53647b1b9b624"
-  """
-  address: String!
-}
-
-enum AccountType {
-  META_CONTRACT
-  POLYJUICE_CONTRACT
-  POLYJUICE_ROOT
-  UDT
-  USER
-}
-
-type AccountUdt {
-  account: Account
-  address_hash: String
-  balance: Decimal
-  id: Int
-  inputs: String
```

Comment:
What is `inputs`? All the types are lack of documentations.

Reply:
error field here,  i deleted it with this pr #395